### PR TITLE
Falco laser fixes + adjustments

### PR DESCRIPTION
**Pull request by Riah321**
Now that ASDI has been implemented we have to fix laser! As an addition, I've added damage falloff similar to P+ to
de-incentivize laser camping, and adjusted the hitboxes to much better match the visual (also similar to P+)

For reference, Melee and P+ both have 12 frames of laser stun completely unstaled and there is no stun falloff. Pre CC rework laser matched this.

First 8 frames of laser = 3% damage and 12F stun unstale
Next 4 frames of laser = 2% damage and 12F stun unstale
Everything after that = 1% damage and 12F stun unstale

Hitboxes slightly repositioned to better match the visual.
1.1 hitlag modifier applies to jablocked opponents.

### CODE REVIEW

**Comment by TeamPearson** [comment]
Yet to play on this but I don't think that laser has had enough time to be hashed out with asdi being merged. My thinking is that asdi being weaker than melee and laser having 1:1 melee values makes this a non necessary change.

**Review by TeamPearson — CHANGES_REQUESTED** [review]
3.0 damage lasers are doing what you want.
2.0 damage lasers are doing what you want.
1.5 lasers need a kbg value of 75 to have 1 less frame of stun to achieve your goal.

**Comment by yoonikoo** [comment]
The changes feel really solid. The further away the opponent is, less hitstun is applied to them from laser. This discourages laser camping as the close laser makes the move committal and rewards aggressive play. This is a middle ground for those that wanted laser nerfed and those that thought that laser was too weak in the current state of the game with the global engine changes.

**Comment by Kaiokendo661** [comment]
Extremely cash money changes 🗿

**Review by TeamPearson — DISMISSED** [review]
If your labbing was correct and that frame data checks out then it lgtm. Not sure why lasers need drop off stun when their utility is net worse than melee and p+ though.

**Comment by Riah321** [comment]
Once the code matches the desired numbers it'll be 1-1. Him not having F1 shine really hurts his defense though so maybe close laser should be 13F? not really sure what to do since that would be extremely degenerate.

**Comment by Riah321** [comment]
PR is tested and finished. Finally.

**Comment by Riah321** [comment]
~~PR is tested and complete. Finally.~~
Turns out it is weight dependent still :D

**Comment by Riah321** [comment]
I've gotten my dev environment fixed, tested on all kinds of weights and it finally, actually does what it says on the tin.

File: fighters/falco/src/acmd/other.rs [hunk 1]
```diff
@@ -187,33 +187,53 @@ unsafe fn falco_blaster_bullet_fly_game(fighter: &mut L2CAgentBase) {
...
     
 }
 
 #[acmd_script( agent = "falco", script = "sound_escapeair" , category = ACMD_SOUND )]
 unsafe fn escape_air_sound(fighter: &mut L2CAgentBase) {
     let lua_state = fighter.lua_state_agent;
     let boma = sv_system::battle_object_module_accessor(lua_state);
-    frame(lua_state, 3.0);
-    if is_excute(fighter) {
+    frame(lua_state, 3.0); {
```
Comment (jobrien97) [inline fighters/falco/src/acmd/other.rs]: This looks mistakenly edited

**Review by jobrien97 — CHANGES_REQUESTED** [review]

**Review by TeamPearson — DISMISSED** [review]
lgtm

File: fighters/falco/src/acmd/other.rs (hunk 1 above)
Comment (TeamPearson) [reply fighters/falco/src/acmd/other.rs]: This appears fixed in the file

**Review by jobrien97 — DISMISSED** [review]

**Review by TeamPearson — DISMISSED** [review]

**Review by TeamPearson — APPROVED** [review]

**Pull request merged**